### PR TITLE
Move Dropdown show/hide logic to HorizontalLinkBar

### PR DESCRIPTION
**Pull request by egreb**
#### Please tick a box ###
- [ ] **feature** _(A new feature_)
- [x] **fix** _(A bug fix_)
- [ ] **refactor** _(A code change that neither fixes a bug nor adds a feature_)
- [ ] **docs** _(Documentation only changes_)
- [ ] **performance** _(A code change that improves performance_)
- [ ] **style** _(Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)_)
- [ ] **build** _(Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)_)
- [ ] **ci** _(Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)_)
- [ ] **test** _(Adding missing tests or correcting existing tests_)

#### If you're adding a feature, is it documented in a storybook story?
- [ ] Yes
- [ ] No

#### Are the components you're working on reusable between brands?
- [ ] Yes _(Using variables that are defined in the default theme)_
- [ ] No _(I hard coded some values)_

#### If you're creating markup, did you add proper semantics? 
- [ ] I added [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)
- [ ] I added microdata from [Schema.org](http://schema.org/)
- [ ] Semantics are derived from HTML
- [ ] Not applicable

_(Did you do a CR and see that there is something that we should check for each PR, that are not on the list, please [update this document](https://github.com/dbmedialab/shiny/edit/master/.github/PULL_REQUEST_TEMPLATE.md))_

Fixes dbmedialab/wolverine-frontend#133

### CODE REVIEW

**Comment by egreb** [comment]
https://github.com/dbmedialab/wolverine-frontend/issues/133

File: src/atoms/LinkBarDropdown.js [hunk 1]
```diff
@@ -18,51 +18,43 @@ const HideMeMaybe = styled.div`
...
+	const {
+		handleClick, id, linkText, children, show,  ...rest
+	} = props;
 
-	handleClick() {
-		this.setState({
-			hide: !this.state.hide,
-		});
-	}
+	const hide  = id !== show;
```
Comment (misund) [inline src/atoms/LinkBarDropdown.js]: There is some double spacing here

**Comment by misund** [comment]
Also, this test should probably not be approved. It doesn't do what it's supposed to do.
https://percy.io/dbmedialab/shiny/builds/637691

**Comment by misund** [comment]
Could it be an option to add an element to the DropDown component that covers the entire page, and has a click listener? (Not saying it's a good idea, just brainstorming)

**Comment by misund** [comment]
Could it be an option to pass the linkbar's ref to the Dropdown component, and close the other Dropdowns from there? (Still just branstorming...)

**Comment by misund** [comment]
How about if the Dropdown component emits an event with its ref in it when it opens, and also listens for the same event - if the ref doesn't match its own, it will close.

**Comment by egreb** [comment]
I don't know what the other suggestions would look like. It seemed logical to me that the nav bar controlled the dropdowns.

**Comment by misund** [comment]
:point_up: I tried an onBlur as an alternative. Let me know what you think.

**Comment by egreb** [comment]
let's go for that. closing this, see https://github.com/dbmedialab/shiny/pull/374 instead

**Pull request closed without merging**